pulsar-functions/instance/src/test/java/org/apache/pulsar/functions/instance/JavaInstanceRunnableTest.java

```diff
@@ -173,6 +178,20 @@ public Void process(String input, Context context) throws Exception {
         }
     }
 
+    @Test
+    public void testFunctionAsyncTime() throws Exception {
+        JavaInstanceRunnable runnable =
+                createRunnable(FunctionDetails.newBuilder().setClassName(TestFunction.class.getName()).build());
+        Instant before = Instant.now();
+        Instant join = CompletableFuture.supplyAsync(() -> {
+            new Thread(runnable).start();
+            return Instant.now();
+        }).join();
+        assertThat(join).isNotNull();
+        Duration duration = Duration.between(before, join);
+        assertThat(duration.toMillis()).isLessThan(20);
```

Comment:
this assertion seems pointless. It would be more meaningful to validate calls to `processTimeEnd`.